### DISCUSSION TITLE
Typo in the documentation

### DISCUSSION THREAD

**Discussion by Pachat** [directus/directus#10987] [Archive]
Is there a way to submit more directly documentation typo's or improvement's?

For instance, in page https://docs.directus.io/configuration/relationships/#translations-o2m it reads 
`The parent collection and primary key are know, so those fields are disabled.`
with an `n` missing at `know`

**Comment by oreilles** [comment]
The docs are open-sourced in this repository: https://github.com/directus/directus/tree/main/docs
PR are welcome if you found some mistakes. 👍
